docs/SVGTemplateRender.md

```diff
@@ -0,0 +1,155 @@
+    {
+      "type": "TemplateRenderMethod",
+      "renderSuite": "svg-mustache",
+      "template": {
+        "id": "https://issuer.example.org/render/membership-card-template.svg",
+        "mediaType": "image/svg+xml"
+      }
+    }
+  ]
+```
+
+**_Example Credential with RenderMethod_**
+
+```json
+{
+  "@context": ["https://www.w3.org/ns/credentials/v2"],
+  "type": ["VerifiableCredential", "MembershipCard"],
+  "issuer": "https://issuer.example.org",
+  "credentialSubject": {
+    "id": "did:example:123",
+    "name": "Mayura Deshmukh",
+    "memberSince": "2023-01-01"
+  },
+  "renderMethod": [
+    {
+      "type": "TemplateRenderMethod",
+      "renderSuite": "svg-mustache",
+      "template": {
+        "id": "https://issuer.example.org/render/membership-card-template.svg",
+        "mediaType": "image/svg+xml"
+      }
+    }
+  ]
+}
+```
+
+## Each Render Method entry describes:
+
+| Field        | Meaning                                                                                 |
+|--------------|-----------------------------------------------------------------------------------------|
+| renderMethod | Describes how the VC should be visually rendered.                                       |
+| type         | Rendering method type (e.g., `TemplateRenderMethod`)                                    |
+| renderSuite  | Defines rendering engine (e.g., `svg-mustache`)                                         |
+| id           | URL pointing to where the SVG template is hosted                                        |
+| mediaType    | Specifies format (`image/svg+xml`)                                                      |
+
+
+## Common Media Types Supported
+
+| MediaType          | Usage                                      |
+|--------------------|---------------------------------------------|
+| text/html          | HTML templates                              |
+| image/svg+xml      | SVG templates (commonly used with Mustache) |
+| application/pdf    | PDF-based renders                           |
+
+
+## What svg-mustache Means
+
+`svg-mustache` is a renderSuite that instructs the verifier:
+
+- “Render this credential using an SVG template that contains Mustache-style placeholders.”
+
+### Components
+
+| Part     | Meaning                                                                                         |
+|----------|-------------------------------------------------------------------------------------------------|
+| SVG      | The template is an SVG graphic (cards, certificates, identity badges).                          |
+| Mustache | Template variables like `{{credentialSubject.name}}` are replaced with actual VC data.          |
+
+# Functionalities 
+
+## How Inji Verify Renders SVG Templates
+
+When a credential is ready to be displayed, Inji Verify performs the following steps:
+
+- `Inji verify UI` fetch the result of the submission through APIs, and validate it using `vc-verifier` and returns the response
```

Comment:
`Inji Verify UI` retrieves the submission result via the provided API. The backend endpoint is responsible for validating the Verifiable Credential (VC) using the vc-verifier library and then returning the validated response to the UI.